src/index.css

```diff
@@ -1,20 +1,37 @@
-
 @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
 
 
 body {
-    font-family: "Poppins", sans-serif  !important;
+    font-family: "Poppins", sans-serif !important;
 }
-.css-ku7d13-MuiTypography-root{
-    font-family: "Poppins", sans-serif  !important;
+
+.css-ku7d13-MuiTypography-root {
+    font-family: "Poppins", sans-serif !important;
 }
-:root{
-    --toastify-color-success:#3252df  !important;
+
+:root {
+    --toastify-color-success: #3252df !important;
     --toastify-icon-color-success: var(--toastify-color-success) !important;
 }
 
 * {
     padding: 0;
     margin: 0;
     box-sizing: border-box;
-  }
+}
+
+.MuiTableCell-root.MuiTableCell-body.MuiTableCell-sizeMedium {
+    color: #3A3A3D !important;
```

Comment:
If we are going to implement dark/light mode functionality, this is not a good practice. As fixed colors, will mean that if we change the MUI theme, this colors will be the same. If we used colors from the MUI theme or created our own theme in MUI,  these colors will change with changing the theme.

Reply:
Also there are better ways to change the styles of the MUI components, you can look at the customization section in MUI.